humanism.sh

```diff
@@ -6,7 +6,13 @@
 #
 # Common aliases
 #
-shopt -s expand_aliases
+
+SH_SOURCE=${BASH_SOURCE:-$_}
```

Comment:
Note that this **must** be the very first instruction, as `$_` is overridden by any previous command